Model/User.php

```diff
@@ -190,9 +176,7 @@ public function unserialize($serialized)
             $this->salt,
             $this->usernameCanonical,
             $this->username,
-            $this->expired,
             $this->locked,
-            $this->credentialsExpired,
```

Comment:
@XWB Sorry, did you make some changes to the `unserialize()` method like I mentioned above? I don't see those changes - I just see the fields removed, but no check for the size of the array and adjustment for the now-missing indexes 4 and 6. Did you adjust something else?